src/public/media.ts

```diff
@@ -391,7 +425,10 @@ export namespace media {
       throw new Error('[select Media] Callback cannot be null');
     }
     ensureInitialized(FrameContexts.content, FrameContexts.task);
-    if (!isAPISupportedByPlatform(mediaAPISupportVersion)) {
+    if (
+      !isAPISupportedByPlatform(mediaAPISupportVersion) ||
```

Comment:
As video is not being planned for web and desktop, please return appropriate error if user is trying selectMedia Video capability on web and desktop

Reply:
Done